### DISCUSSION TITLE
How to share the object between two routes. or is there any other way to achieve the following use case in Fastapi

### DISCUSSION THREAD

**Discussion by SAIVENKATARAJU** [fastapi/fastapi#6595] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@data.post("/readfileandstore/")
async def preprocess_file(file:UploadFile = File(...)):
     if 'txt' in file.filename:
         pipe_document=PreProcess.store_txt_data(file)
@data.post("/findanswer")
async def find_answer(data:dict):
     answers=[]
     question = data['question']
     logging.info(question)
     prediction_document = pipe_document.run(query=question, top_k_retriever=128, top_k_reader=2)
     answers.append(post_processing(prediction_document))
     return answers
```

### Description

I am trying to create a webapp which allows to upload users a file and can perform search and QnA on the same file. In the route "readfileandstore" , I am taking file from users and sending to other functions for futher steps. as this is a one time process, I am using one route.  after , I need Pipe_document object to process further quires on the document. so is there any way to get the pipe_document in the second route.

    
   

### Operating System

Linux

### FastAPI Version

Na

### Python Version

na

**Comment by YuriiMotov** [comment, marked as answer]
No, you should store this object somewhere on the server and then access it using unique key from second endpoint
